sql-files/upgrades/2017-03-02--11-40.sql

```diff
@@ -0,0 +1,105 @@
+	ADD COLUMN `opt_idx3` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val2`,
+	ADD COLUMN `opt_val3` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx3`,
+	ADD COLUMN `opt_idx4` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val3`,
+	ADD COLUMN `opt_val4` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx4`;
+	
+ALTER TABLE `inventory`
+	ADD COLUMN `opt_idx0` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `card3`,
+	ADD COLUMN `opt_val0` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx0`,
+	ADD COLUMN `opt_idx1` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val0`,
+	ADD COLUMN `opt_val1` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx1`,
+	ADD COLUMN `opt_idx2` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val1`,
+	ADD COLUMN `opt_val2` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx2`,
+	ADD COLUMN `opt_idx3` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val2`,
+	ADD COLUMN `opt_val3` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx3`,
+	ADD COLUMN `opt_idx4` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val3`,
+	ADD COLUMN `opt_val4` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx4`;
+	
+ALTER TABLE `mail`
+	ADD COLUMN `opt_idx0` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `card3`,
+	ADD COLUMN `opt_val0` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx0`,
+	ADD COLUMN `opt_idx1` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val0`,
+	ADD COLUMN `opt_val1` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx1`,
+	ADD COLUMN `opt_idx2` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val1`,
+	ADD COLUMN `opt_val2` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx2`,
+	ADD COLUMN `opt_idx3` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val2`,
+	ADD COLUMN `opt_val3` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx3`,
+	ADD COLUMN `opt_idx4` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val3`,
+	ADD COLUMN `opt_val4` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx4`;
+	
+ALTER TABLE `storage`
+	ADD COLUMN `opt_idx0` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `card3`,
+	ADD COLUMN `opt_val0` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx0`,
+	ADD COLUMN `opt_idx1` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val0`,
+	ADD COLUMN `opt_val1` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx1`,
+	ADD COLUMN `opt_idx2` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val1`,
+	ADD COLUMN `opt_val2` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx2`,
+	ADD COLUMN `opt_idx3` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val2`,
+	ADD COLUMN `opt_val3` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx3`,
+	ADD COLUMN `opt_idx4` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_val3`,
+	ADD COLUMN `opt_val4` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0' AFTER `opt_idx4`;
+
+ALTER TABLE `picklog`
```

Comment:
When a migration affects the main database and the logs database, it should always be split into two, so that people can easily run the two scripts separately when they have separate databases.